Add note regarding cypress env vars (#581)
saucelabs/sauce-docs: docs/testrunner-toolkit/configuration/cypress.md

      hello: world
      my_var: $MY_VAR
```

> :::note
> Environment variables are directly provided to Cypress. You should not use `CYPRESS_` prefix.
> :::

Reviewer: ```suggestion
:::caution
Since environment variables are provided to Cypress directly, avoid using `CYPRESS_` as a prefix.
:::
```

Author: @sweeneyskirt-sl I made sure that even if they add the `CYPRESS_` this will be still working. That has been made so we do not break any existing customer configuration.
That being said, this is not the way it's expected to be done, and may be removed in future versions.

It also applies to global `env` settings, as both of them are merged before invoking cypress.

Reviewer: Perfect. Could you please add the caution to the global `env` description as well?

Author: Sure !
